pkg/cloud/vsphere/services/govmomi/extra/config.go

```diff
@@ -27,7 +27,7 @@ type Config []types.BaseOptionValue
 
 // SetCloudInitUserData sets the cloud init user data at the key
 // "guestinfo.userdata" as a base64-encoded string.
-func (e *Config) SetCloudInitUserData(data []byte) error {
+func (e *Config) SetCloudInitUserData(data []byte) {
```

Comment:
Hi @vijaykatam,

Please remove this and the other function signature change as they are not integral to this patch. Feel free to open a separate PR for the functions. Thank you!